CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 1.0.8.2 (2019-09-23)
+## Edge Hub
+### Bug Fixes
+* Set RocksDB total WAL size to 512M ([07ee35d](https://github.com/Azure/iotedge/commit/07ee35d79a11eb6daab2a883e82ffbd7a1e93682))
+* Fix leaking event handlers in offline scenario ([f3b8a46](https://github.com/Azure/iotedge/commit/f3b8a463372d94163c286bd4c1a81ea9eb811832))
+* Fix error if client does not send product info  ([9267a05](https://github.com/Azure/iotedge/commit/9267a0545ac4394400afbcabdc123737ebcfbc17))
+
```

Comment:
This block is already in 1.0.8 and appears due to a prior merge. See [here](https://github.com/Azure/iotedge/blob/release/1.0.8/CHANGELOG.md)